diracx/values.yaml

```diff
@@ -51,7 +51,8 @@ podSecurityContext:
   # fsGroup: 2000
 
 securityContext:
-  {}
+  fsGroup: 2000
```

Comment:
That looks good ! I am just not sure whether this should be the default value or if empty should be the default. I will try to test it on openshift before merging it

Reply:
2000 comes from the user internal to the chart/images, so I thought it should be inside.